tests/webview/ui/test_content.py

```diff
@@ -766,3 +766,36 @@ def test_chapter_review_version_matches_book_version(webview_base_url, selenium,
 
     # THEN the chapter review version matches the book version
     assert content.page_version == content.book_version
+
+
+@markers.webview
+@markers.test_case('C195071')
+@markers.nondestructive
+@markers.parametrize('id', ['yWshIYVW@1'])
+def test_buggy_msg_not_show_in_left_panel(webview_base_url, id, selenium):
+    # GIVEN the webview base url, a chapter page id, and the Selenium driver
+
+    # WHEN we visit that page of the chapter
+    content = Content(selenium, webview_base_url, id=id).open()
+
+    # THEN correct book info is listed in the left panel
+    if content.is_left_panel_title_displayed:
+        assert content.is_go_to_book_link_present
+
+
+@markers.webview
+@markers.test_case('C195061')
+@markers.nondestructive
+@markers.parametrize('id', ['BWYBGK7C@2'])
+def test_book_title_not_limited(webview_base_url, id, selenium):
+    # GIVEN the webview base url, a chapter page id, and the Selenium driver
+
+    # WHEN we visit that page of the chapter
+    content = Content(selenium, webview_base_url, id=id).open()
+
+    # THEN get the title text to check if it matches what it suppose to be
+    actual_title = content.get_left_nav_book_title
+    correct_title = '"Our Cultural Commonwealth" The Report of the American ' \
+                    'Council of Learned Societies Commission on Cyberinfrastructure ' \
+                    'for the Humanities and Social Sciences'
+    assert actual_title == correct_title
```

Comment:
We do not want a hardcoded title. When something is hardcoded in ANY code you always want to question what you are doing. The test case does suggest that what you want to do is compare the title text but as revealed in the discussion #228 what we actually want to is to test that the ellipses `...` is not in the title.

I believe that would look something like this:
`assert '...' not in book_title`